StripeUICore/StripeUICore/Source/Helpers/LiquidGlassHelpers.swift

```diff
@@ -31,3 +33,24 @@ import Foundation
         return false
     }
 }
+
+// MARK: - UIView Liquid Glass helpers
+@_spi(STP) extension UIView {
+    @_spi(STP) public func ios26_applyCapsuleCornerConfiguration() {
+        stpAssert(LiquidGlassDetector.isEnabled)
+#if canImport(DeclaredAgeRange) // Hack to check if we're compiling w/ the iOS 26 SDK
```

Comment:
interesting ...any reason to use this over the standard  if compiler check?

Reply:
realized we can't use that one b/c we still use swift 5.

Reply:
i'm wrong - we use xcode's compiler in swift 5 language mode.